index.html

```diff
@@ -65,11 +65,11 @@
   local draft_filter="cat"
   local url
   case ${VERSION} in
-    rc) url=https://api.github.com/repos/submariner-io/submariner-operator/releases
+    rc) url=https://api.github.com/repos/submariner-io/releases/releases
            draft_filter="grep \-rc"
            ;;
-    latest) url=https://api.github.com/repos/submariner-io/submariner-operator/releases/latest ;;
-    *) echo "https://github.com/submariner-io/submariner-operator/releases/download/${VERSION}/subctl-${VERSION}-${os}-${architecture}.tar.xz"
+    latest) url=https://api.github.com/repos/submariner-io/releases/releases/latest ;;
+    *) echo "https://github.com/submariner-io/releases/releases/download/${VERSION}/subctl-${VERSION}-${os}-${architecture}.tar.xz"
```

Comment:
Versions older than 0.8.0 should use the old url (can be hardcoded in the `case` i guess)